packages/arb-token-bridge-ui/src/components/TransferPanel/useTransferReadiness.ts

```diff
@@ -366,93 +281,109 @@ export function useTransferReadiness({
       // No error while loading gas costs
       case 'idle':
       case 'loading':
-        return undefined
+        return notReady()
 
-      case 'error':
-        return TransferPanelMainRichErrorMessage.GAS_ESTIMATION_FAILURE
+      case 'unavailable':
+        return ready()
 
-      case 'success': {
-        const sanitizedEstimatedGasFees = sanitizeEstimatedGasFees(gasSummary, {
-          isSmartContractWallet,
-          isDepositMode
+      case 'error':
+        return notReady({
+          errorMessage:
+            UseTransferReadinessRichErrorMessage.GAS_ESTIMATION_FAILURE
         })
 
-        const defaultRequiredGasFees =
-          sanitizedEstimatedGasFees.estimatedL1GasFees +
-          sanitizedEstimatedGasFees.estimatedL2GasFees
+      case 'success': {
```

Comment:
not so sure about this
the `success` case is long enough to be abstracted into a function itself

Reply:
Fair, but I can do that in another PR, this just combines what we already had